### PR TITLE
You can't miss blinding yourself with a laser pointer

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Comment by Buggy123** [comment]
Is that a challenge?

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -96,7 +96,7 @@
 				severity = 0
 
 			//chance to actually hit the eyes depends on internal component
-			if(prob(effectchance * diode.rating) && C.flash_act(severity))
+			if((C == user || prob(effectchance * diode.rating)) && C.flash_act(severity))
 				outmsg = "<span class='notice'>You blind [C] by shining [src] in their eyes.</span>"
```
Comment (vuonojenmustaturska) [inline code/game/objects/items/devices/laserpointer.dm]: Can you make the message reflect the fact that you specifically blinded yourself and not another guy with your name

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/devices/laserpointer.dm]: make people with the same name have 100% blind chance instead :^)

**Comment by Cyberboss** [comment]
Too lazy

**Pull request closed without merging**